shared/dto/widgets/page-filter.dto.ts

```diff
@@ -0,0 +1,9 @@
+export type PageFilter = {
+  name: string;
+  isCustomFilter: boolean;
+  values: string[];
+  label: {
+    selected: string;
+    unSelected: string;
+  };
+};
```

Comment:
proposal to change this to:
`type Filter = {
  name: string;
  values: string[];
  label: DefaultFilterLabels | string
};

type DefaultFilterLabels = {
  selected: string;
  unselected: string;
};`

Reply:
Let's go into more details as we are able to pull actual data from somewhere